devinfo/src/mountinfo/io_utils.rs

```diff
@@ -0,0 +1,27 @@
+use crate::mountinfo::error::{MountInfoError, Result};
+use std::{fs::read, path::Path};
+
+const DEFAULT_RETRY_COUNT: u32 = 2;
+
+pub(crate) fn consistent_read<P: AsRef<Path>>(
+    path: P,
+    retry_count: Option<u32>,
+) -> Result<Vec<u8>> {
+    let mut current_content = read(path.as_ref())?;
+
+    let retries = retry_count.unwrap_or(DEFAULT_RETRY_COUNT);
+    for _ in 0 .. retries {
```

Comment:
Otherwise there's no check? Or was this intended?

```suggestion
    for _ in 0 ..= retries {
```

Reply:
This is intentional. Taking an example: retries=2,
0..= retries goes from 0 to 2, which is 3 iterations
0.. retries goes to 0 to 1, which is 2 iterations, which is what we want when we say 2 retries.

Does this make sense? Or is retries in the sense that the usual try and then the retries, then it comes to 3.

Reply:
I've renamed the variable to 'attempts'.

Reply:
> Otherwise there's no check
Oh I get your point. Yeah, needs at least 1. Needs '='.

Reply:
Renamed back to retries, makes more sense now with the '='.

Reply:
> > Otherwise there's no check
> 
> Oh I get your point. Yeah, needs at least 1. Needs '='.

Yeah, for retries = 2:
0..=2
0: compare first time
1: retry n1
2: retry n2